docs/dev/run_the_etl.rst

```diff
@@ -0,0 +1,88 @@
+.. _run-the-etl:
+
+=======================
+Walk, Don't Run the ETL
+=======================
+
+So you want to run the PUDL data processing pipeline? This is the most involved
+way to get access to PUDL data. It is only suggested if you want to edit the
+ETL process or contribute to the code base. Check the basic usage page for the
+other access methods (see: :ref:`basic-usage`). Measure twice, cut once.
+
+These instructions assume you have already gone through the development setup
+(see: :ref:`dev_setup`).
+
+There are four main scripts that are involved in the PUDL processing pipeline:
+
+1. `ferc1_to_sqlite` :doc: clones FERC Form 1's dbf files <clone_ferc1> as
+   a sqlite database - a necessary prep extract step for FERC Form 1.
+2. `pudl_etl` is where the magic happens. This is the main script which
+   coordinates the "Extract, Transform, Load" steps which results in
+   `Tabular Data Packages <https://frictionlessdata.io/specs/tabular-data-package/>`_.
+3. `datapkg_to_sqlite` converts the Tabular Data Packages into a SQLite
+   database. We recommend this for all of the small-medium tables which is
+   currently everything but the hourly EPA CEMS data.
+4. `epacems_to_parquet` converts the Tabular Data Packages into parquet files
+   for the large datasets - which is currently the hourly EPA CEMS data.
```

Comment:
This reads a big ambiguously to me -- like it could in theory convert other data packages into Parquet -- but this is very much a special purpose epacems only script.